client/configuration/webpack/webpack.config.prod.js

```diff
@@ -56,11 +56,8 @@ const prodConfig = {
       {
         test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2|otf)$/i,
         loader: "file-loader",
-        include: [nodeModules, fonts],
         options: {
           name: "static/assets/[name]-[contenthash].[ext]",
-          // (thuang): This is needed to make sure @font url path is '../static/assets/'
-          publicPath: "..",
```

Comment:
Did you test this change? @sienacizdziel and I made this change after trying to get fonts to work for a few hours

Reply:
I tested it by running `make build-for-server-dev && make start-server` and visit `http://localhost:5005/d/pbmc3k.cxg/`, and the static assets path match what dev/staging env have!

Reply:
More details here: https://github.com/chanzuckerberg/single-cell-explorer/pull/391

Reply:
context for keeping Roboto Condensed in the meantime:

Harley: "Inter takes up more horizontal space, so gene names were getting truncated with fewer characters. I still think we should switch to Inter, but I’d like to make a couple improvements to the gene UX to make more space for the gene name (see screenshot). Until we make those changes, Arathi wanted to revert back to Roboto Condensed."

<img width="1440" alt="Explorer" src="https://user-images.githubusercontent.com/6309723/189437682-77e77632-7fb3-4152-b684-1014cbbef3f0.png">